sw/device/lib/testing/flash_ctrl_testutils.c

```diff
@@ -400,3 +400,22 @@ void flash_ctrl_testutils_backdoor_wait_update(
       &data, kDifFlashCtrlPartitionTypeData, 1));
   IBEX_SPIN_FOR(UINT32_MAX != *(uint32_t *)addr, timeout);
 }
+
+void flash_ctrl_testutils_backdoor_wait_eq(dif_flash_ctrl_state_t *flash_state,
```

Comment:
We don't need this anymore, at least it seems so. But I propose to keep it because it is a very useful function.